generative/losses/perceptual.py

```diff
@@ -65,8 +79,15 @@ def __init__(
             self.perceptual_function = MedicalNetPerceptualSimilarity(net=network_type, verbose=False)
```

Comment:
My further thought is that maybe in the next version, consider the network as an arg directly rather than instantiate it within this class is a better way for users to define their customized network for feature extraction (then calculate the loss).

Reply:
Hi @Warvito @ericspod , what do you think?

Reply:
We should have a facility to allow users to provide their own networks yes. We could have an argument to the constructor `perceptual_function` which if not `None` will be used as the `self.perceptual_function` value, otherwise the existing choices will be made.

Reply:
Thanks @ericspod !